doc/build/orm/basic_relationships.rst

```diff
@@ -263,55 +272,64 @@ remote tables with which to link::
     this ensures that duplicate rows won't be persisted within the table regardless
     of issues on the application side::
 
-        association_table = Table('association', Base.metadata,
-            Column('left_id', ForeignKey('left.id'), primary_key=True),
-            Column('right_id', ForeignKey('right.id'), primary_key=True)
-        )
+    association_table = Table(
+        "association",
+        Base.metadata,
+        Column("left_id", ForeignKey("left.id"), primary_key=True),
+        Column("right_id", ForeignKey("right.id"), primary_key=True),
+    )
```

Comment:
I think this should have an additional indentation for rst to consider it code

Reply:
That's probably the case, i'm not really familiar with rst, i will restore original indentation